pandas/indexes/base.py

```diff
@@ -1977,6 +2009,36 @@ def symmetric_difference(self, other, result_name=None):
 
     sym_diff = deprecate('sym_diff', symmetric_difference)
 
+    def _get_unique_index(self, dropna=False):
+        """
+        Returns an index containing unique values.
+
+        Parameters
+        ----------
+        dropna : bool
+            If True, NaN values are dropped.
```

Comment:
Actually, I no longer need a `dropna` parameter, but decided to keep it for now.